openpype/plugins/publish/collect_otio_subset_resources.py

```diff
@@ -225,3 +237,26 @@ def _create_representation(self, start, end, **kwargs):
         if kwargs.get("trim") is True:
             representation_data["tags"] = ["trim"]
         return representation_data
+
+    def get_template_name(self, instance):
```

Comment:
This is theorectical question which probably does not have solution at this moment:

Template should be calculated based on instance data, not on context data. For example during TrayPublisher publishing `context.data['anatomyData']` contains only project data (because there is not a global asset or task) -> and I know that this is not plugin which runs on traypublisher but still.

Additional theorectical question:
Is this is needed? Why is neededed to change of frames on version data here? I don't know where the files exactly are comming from but they could come from extraction? Could frame fill for version data rather happen during extraction?

Reply:
I get your point. I would like to have rather instance anatomy data then context but the order is wrong here. I will try to shift order of this plugin after `collect_anatomy_instance_data.py` and see if it works. 

Regarding necesity - unfortunatelly there is not other way to find out if I should use media framerange for version or workfile framerange. This should have been done based on the condition if `"{originalBasename}"` is in template path and only way to find out is to get template set for the host/family presets.